modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/components/detail/SocialDetail.js

```diff
@@ -42,6 +42,7 @@ const SOCIAL_MEDIA_COLORS = {
 
 export default function SocialDetail({
 	currentPage,
+	hideTimeSpanSelector = true,
```

Comment:
Not a huge deal, and this might just be my personal preference, but if we have boolean props to hide/show, I typically think it's better to be "opt-in" rather than "opt-out".

For example:
```jsx
// Current implementation
<SocialDetail />
<SocialDetail hideTimeSpanSelector={false} />

// Opt-in props
<SocialDetail />
<SocialDetail showTimeSelector />
```

It's more of a small readability difference. I like to think through how I would read it out=loud

`hideTimeSpanSelector={false}` = "Don't hide the time selector". 
`showTimeSelector` = "Show time selector". 


IMO, I think the opt-in is easier to read and I don't have to think about double negations "false hide".